README.md

```diff
@@ -1,15 +1,56 @@
-This repository has two seperate exercises depending on the role:
+# SingleStone Python Data Exercises
+
+These exercises are designed to show us how you approach writing software that will
+be used and maintained by others. They are to be written in Python, but aren't 
+intended to demonstrate a deep knowledge of that language or any particular tooling. 
+
+## General Instructions
+
+When completing an exercise, please focus on the following:
+
+* Write code for other people. What will happen when (not if) you rotate off this
+project and someone else has to maintain your code?
+* Make the code easy to deploy anywhere. Assume the person grading this has no more
+than 5 minutes to get your code running.
+* Embrace constraints, but assume they are not static. Use design patterns that 
+make your code easy to change. 
+
+In particular, we would like to see:
+1. A README that explains to a **non-programmer** what the software does and 
```

Comment:
Its a little wordy.

Reply:
I'll give it another pass, but we want to err on the side of clarity about what were evaluating them on. This is as much about what we value as it is about their technical skill.